package.json

```diff
@@ -15,7 +15,7 @@
     "coverage": "nyc report --reporter=text-lcov | coveralls && nyc report --reporter=lcov",
     "clean": "rm -rf dist && mkdir dist",
     "build": "npm run clean && npm run babel-build",
-    "heroku-postbuild": "npm run build && export NODE_ENV=production && npm start",
+    "heroku-postbuild": "npm run build && export NODE_ENV=production && npm run migration && npm start",
```

Comment:
Why are we changing this here?

Reply:
I wasn't the one that did that, I just rebased and saw that